admin_manual/release_notes.rst

```diff
@@ -31,6 +31,11 @@ value; if there is not one then use empty quotes, ``""``. (See
 Users of the Linux Package needs to update their repository setup like described
 in this `blogpost <https://owncloud.org/blog/upgrading-to-owncloud-server-8-2/>`_.
 
+The log format for the failed logins to the ``owncloud.log`` got a slightly syntax 
```

Comment:
"for the failed logins" => "for failed logins"
"got a slightly syntax bugfix" => "has a slight syntax change"